src/ngx_http_lua_control.c

```diff
@@ -319,11 +319,18 @@ ngx_http_lua_ngx_exit(lua_State *L)
                                | NGX_HTTP_LUA_CONTEXT_CONTENT
                                | NGX_HTTP_LUA_CONTEXT_TIMER
                                | NGX_HTTP_LUA_CONTEXT_HEADER_FILTER
+                               | NGX_HTTP_LUA_CONTEXT_BODY_FILTER
                                | NGX_HTTP_LUA_CONTEXT_BALANCER
                                | NGX_HTTP_LUA_CONTEXT_SSL_CERT
                                | NGX_HTTP_LUA_CONTEXT_SSL_SESS_STORE
                                | NGX_HTTP_LUA_CONTEXT_SSL_SESS_FETCH);
 
+    /* rc is useless in bodyfilter phase */
```

Comment:
We come across a situation where we need to constantly compare a flag to determine whether or not to stop the output stream, so we need to read the flag in `body_filter` and stop it. At this point because the Http response header has been sent, so I think `rc` is not too much significance